### PR TITLE
Adopt Compose v2

### PR DESCRIPTION
**Pull request by ndeloof**
Prefer `docker compose` command as Compose v2 is now G.A and Compose v1 will be deprecated in a few
Also prefer `compose.yaml` to embrace the compose specification

### CODE REVIEW

File: django/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Project structure:
 ```
 .
-├── docker-compose.yml
+├── docker compose.yml
```
Comment (glours) [inline django/README.md:7]:
```suggestion
├── compose.yml
```

File: elasticsearch-logstash-kibana/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 Project structure:
 ```
 .
-└── docker-compose.yml
+└── docker compose.yml
```
Comment (glours) [inline elasticsearch-logstash-kibana/README.md:7]:
```suggestion
└── compose.yml
```

File: elasticsearch-logstash-kibana/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
...
 ```
 
 [_docker-compose.yml_](docker-compose.yml)
```
Comment (glours) [inline elasticsearch-logstash-kibana/README.md:10]:
```suggestion
[_compose.yml_](compose.yml)
```

File: nextcloud-postgres/README.md [hunk 4]
```diff
@@ -24,10 +24,10 @@ services:
...
 
-When deploying this setup, docker-compose maps the nextcloud container port 80 to
+When deploying this setup, docker compose maps the nextcloud container port 80 to
 port 80 of the host as specified in the compose file.
 
-## Deploy with docker-compose
+## Deploy with docker compose
 
 ```
 $ docker$ compose up -d
```
Comment (glours) [inline nextcloud-postgres/README.md:33]:
```suggestion
$ docker compose up -d
```

File: nextcloud-redis-mariadb/README.md [hunk 5]
```diff
@@ -29,10 +29,10 @@ services:
...
 
-When deploying this setup, docker-compose maps the nextcloud container port 80 to
+When deploying this setup, docker compose maps the nextcloud container port 80 to
 port 80 of the host as specified in the compose file.
 
-## Deploy with docker-compose
+## Deploy with docker compose
 
 ```
 $ docker$ compose up -d
```
Comment (glours) [inline nextcloud-redis-mariadb/README.md:38]:
```suggestion
$ docker compose up -d
```

File: nginx-nodejs-redis/README.md [hunk 6]
```diff
@@ -6,7 +6,7 @@ Project structure:
 ```
 .
 ├── README.md
-├── docker-compose.yml
+├── docker compose.yml
```
Comment (glours) [inline nginx-nodejs-redis/README.md:9]:
```suggestion
├── compose.yml
```

File: django/README.md [hunk 7]
```diff
@@ -4,7 +4,7 @@
 Project structure:
```
Comment (glours) [inline django/README.md:4]: Missing the rename of the docker-compose.yaml file

File: elasticsearch-logstash-kibana/README.md [hunk 8]
```diff
@@ -4,7 +4,7 @@
 Project structure:
```
Comment (glours) [inline elasticsearch-logstash-kibana/README.md:4]: Missing the rename of the docker-compose.yaml for this sample

File: fastapi/README.md [hunk 9]
```diff
@@ -3,7 +3,7 @@
```
Comment (glours) [inline fastapi/README.md:3]: Missing the rename of the docker-compose.yaml for this sample

File: flask-redis/README.md [hunk 10]
```diff
@@ -9,7 +9,7 @@ Project structure:
 ├── Dockerfile
```
Comment (glours) [inline flask-redis/README.md:9]: Missing the rename of the docker-compose.yaml for this sample

File: minecraft/README.md [hunk 11]
```diff
@@ -4,11 +4,11 @@ This example defines a basic setup for a Minecraft server. More details on the M
 Project structure:
```
Comment (glours) [inline minecraft/README.md:4]: Missing the rename of the docker-compose.yaml for this sample

File: nginx-golang/README.md [hunk 12]
```diff
@@ -7,14 +7,14 @@ Project structure:
 ├── backend
```
Comment (glours) [inline nginx-golang/README.md:7]: Missing the rename of the docker-compose.yaml for this sample

File: nginx-nodejs-redis/README.md [hunk 13]
```diff
@@ -6,7 +6,7 @@ Project structure:
 ```
```
Comment (glours) [inline nginx-nodejs-redis/README.md:6]: Missing the rename of the docker-compose.yaml for this sample

File: prometheus-grafana/README.md [hunk 14]
```diff
@@ -4,7 +4,7 @@
 Project structure:
```
Comment (glours) [inline prometheus-grafana/README.md:4]: Missing the rename of the docker-compose.yaml for this sample

File: react-express-mongodb/README.md [hunk 15]
```diff
@@ -7,14 +7,14 @@ Project structure:
 ├── backend
```
Comment (glours) [inline react-express-mongodb/README.md:7]: Missing the rename of the docker-compose.yaml for this sample

File: react-nginx/README.md [hunk 16]
```diff
@@ -5,7 +5,7 @@
 Project structure:
```
Comment (glours) [inline react-nginx/README.md:5]: Missing the rename of the docker-compose.yaml for this sample

File: traefik-golang/README.md [hunk 17]
```diff
@@ -7,11 +7,11 @@ Project structure:
 ├── backend
```
Comment (glours) [inline traefik-golang/README.md:7]: Missing the rename of the docker-compose.yaml for this sample

File: prometheus-grafana/README.md [hunk 18]
```diff
@@ -4,7 +4,7 @@
...
 ```
 .
-├── docker-compose.yml
+├── docker compose.yml
```
Comment (glours) [inline prometheus-grafana/README.md:7]:
```suggestion
├── compose.yml
```

File: react-nginx/README.md [hunk 19]
```diff
@@ -5,7 +5,7 @@
...
 
 ```
-├── docker-compose.yml
+├── docker compose.yml
```
Comment (glours) [inline react-nginx/README.md:8]:
```suggestion
├── compose.yml
```

File: traefik-golang/README.md [hunk 20]
```diff
@@ -7,11 +7,11 @@ Project structure:
...
 │   ├── Dockerfile
 │   └── main.go
-├── docker-compose.yml
+├── docker compose.yml
```
Comment (glours) [inline traefik-golang/README.md:10]:
```suggestion
├── compose.yml
```

**Review by glours — CHANGES_REQUESTED** [review]
Some minor changes to address
Thanks @ndeloof for this contribution 🤗

File: react-nginx/README.md (hunk 16 above)
Comment (ndeloof) [reply react-nginx/README.md:5]: indeed, I searched for docker-compose.yaml, not docker-compose.yml

**Review by glours — APPROVED** [review]
LGTM

File: traefik-golang/README.md (hunk 20 above)
Comment (shahgulshahgul93-pixel) [reply traefik-golang/README.md]: Update

**Pull request merged**